Description: fix for a spelling mistake
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2014-06-26
Forwarded: https://github.com/sanko/readonly/issues/7

--- a/lib/Readonly.pm
+++ b/lib/Readonly.pm
@@ -663,7 +663,7 @@
 internal methods C<SvREADONLY> and C<SvREADONLY_on> to lock simple scalars. On
 the surface, everything was peachy but things weren't the same behind the
 scenes. In edge cases, code perfromed very differently if Readonly::XS was
-installed and because it wasn't a required dependancy in most code, it made
+installed and because it wasn't a required dependency in most code, it made
 downstream bugs very hard to track.
 
 In the years since Readonly::XS was released, the then private internal
@@ -676,7 +676,7 @@
 
 =item * You do not need to install Readonly::XS.
 
-=item * You should stop listing Readonly::XS as a dependancy or expect it to
+=item * You should stop listing Readonly::XS as a dependency or expect it to
 be installed.
 
 =item * Stop testing the C<$Readonly::XSokay> variable!
